### PR TITLE
page: user profile page added

### PR DESCRIPTION
**Pull request by manojsarna**
- [x] User Profile Page Added

### Ignore Rest of the Files

image
image

### CODE REVIEW

File: src/pages/user/User.jsx [hunk 1]
```diff
@@ -1,5 +1,53 @@
+import { useDispatch, useSelector } from "react-redux";
+import { useDocTitle } from "../../hooks/useDocTitle";
+import { logout } from "../../redux/reducers/authSlice";
+import { InfinitySpin as Loader } from "react-loader-spinner";
 import "./user.css";
 
 export function User() {
-  return <main className="sm-main"></main>;
+  const { loading, user } = useSelector((state) => state.auth);
+  console.log(loading, user);
```
Comment (pankaj-prog) [inline src/pages/user/User.jsx]: remove console

File: src/pages/user/User.jsx [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+    </div>
+  ) : (
+    <main className="sm-main">
+      <div className="sm-main-user-profile-container">
+        <div className="sm-main-user-profile">
+          <p className="sm-main-heading">
+            {`${user?.firstName[0].toUpperCase()}${user?.lastName[0].toUpperCase()}'s `}
+            Profile Page
+          </p>
+        </div>
```
Comment (pankaj-prog) [inline src/pages/user/User.jsx:25]: use semantic tag, as here you can use header

**Review by pankaj-prog — APPROVED** [review]

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: src/App.jsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ function App() {
         <Route path="/mockman" element={<MockAPI />} />
 
         <Route element={<PrivateRoute />}>
-          {/* <Route path="/user" element={<User />} /> */}
+          <Route path="/user" element={<User />} />
         </Route>
```
Comment (tarunsankhla) [inline src/App.jsx:29]: Use constant not direct string

File: src/pages/user/User.jsx [hunk 4]
```diff
@@ -1,5 +1,52 @@
...
+        <div className="sm-user-profile-info">
+          <div className="sm-user-card">
+            <div className="avatar avatar-hover avatar-text bg-3 s-xxl">
+              <h2>{`${user?.firstName[0].toUpperCase()}${user?.lastName[0].toUpperCase()}`}</h2>
+            </div>
+            <div className="sm-user-content">
+              <h2>{`${user?.firstName} ${user?.lastName}`}</h2>
```
Comment (tarunsankhla) [inline src/pages/user/User.jsx:32]: Why template literals here

**Review by tarunsankhla — APPROVED** [review]
Over all Good just minor changes

File: src/pages/user/User.jsx (hunk 4 above)
Comment (manojsarna) [reply src/pages/user/User.jsx:32]: for spacing

**Pull request merged**